aten/src/ATen/native/native_functions.yaml

```diff
@@ -166,15 +166,16 @@
 - func: _debug_has_internal_overlap(Tensor self) -> int
   variants: function
 
-- func: _fused_dropout(Tensor self, float p, Generator? generator=None) -> (Tensor, Tensor)
+- func: native_dropout(Tensor input, float p, float scale, bool train) -> (Tensor, Tensor)
   variants: function
   dispatch:
-    CUDA: fused_dropout_cuda
+    CPU: native_dropout_cpu
+    CUDA: native_dropout_cuda
```

Comment:
consider making this structured!

Reply:
Can you point me to some docs on that? I didn't find it under https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/README.md

I can mimic some porting PRs like #57619.